tests/container_disk_test.go

```diff
@@ -63,7 +63,7 @@ var _ = Describe("[rfe_id:588][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 
 	Describe("[rfe_id:273][crit:medium][vendor:cnv-qe@redhat.com][level:component]Starting and stopping the same VirtualMachine", func() {
 		Context("with ephemeral registry disk", func() {
-			It("[test_id:1463][Conformance] should success multiple times", decorators.Conformance, func() {
+			It("[test_id:1463] should success multiple times", decorators.Conformance, func() {
```

Comment:
@akalenyu @ShellyKa13 are you comfortable with this change?

Reply:
worth to mention same was done on main
https://github.com/kubevirt/kubevirt/pull/14509/commits/e160006e3a7157898646cbb960d45a4095a6d46a

Reply:
this is a sig-compute test actually

Reply:
Thank you @akalenyu!